dev/group_vars/apim-analytics.yml

```diff
@@ -88,7 +88,7 @@ message_tracing_db_password: wso2carbon
 message_tracing_db_driver: org.h2.Driver
 message_tracing_db_test_query: SELECT 1
 
-apim_analytics_db_url: jdbc:h2:${sys:carbon.home}/wso2/worker/database/WSO2AM_STATS_DB;AUTO_SERVER=TRUE
+apim_analytics_db_url: jdbc:h2:${sys:carbon.home}/../../apim-analytics-worker/wso2am-analytics-3.2.0/wso2/database/WSO2AM_STATS_DB;AUTO_SERVER=TRUE
```

Comment:
Shall we parameterize the version in `wso2am-analytics-3.2.0` here? Could be an area someone might miss when modifying the Ansible resources for a new release.

Reply:
Fixed with e79968f8fe8f2c69db8216cb1d4d5137559e4b3b